Description: build s390 debugging symbols with -gstabs instead of with -g
             to reduce linker memory usage.
             Thanks to Oswald Buddenhagen for the hint.
author: Sune Vuorela <sune@debian.org>
Bug-Debian: http://bugs.debian.org/528485

--- a/WebKit.pri
+++ b/WebKit.pri
@@ -59,6 +59,13 @@ CONFIG(release, debug|release) {
     DEFINES += NDEBUG
 }
 
+equals(QT_ARCH, s390) {
+	message("S/390 workaround for QtWebkit: do not build with -g, but with -gstabs")
+	QMAKE_CXXFLAGS -= -g
+	QMAKE_CXXFLAGS += -gstabs
+}
+
+
 BASE_DIR = $$PWD
 
 symbian {
